README.md

```diff
@@ -65,6 +65,45 @@ app('api.router')->version(config('api.version'), ['namespace' => 'Saritasa\Lara
   });
 ```  
 
```

Comment:
```suggestion
### Customize login request
If you need to send other credentials, then `email` and `password`
```

Header - what example case in README is described.

Reply:
done